src/components/events/Modal.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { FaCircle } from "react-icons/fa";
+import { AiOutlinePlus } from "react-icons/ai";
+import Link from "next/link";
+
+const ListElement = ({ innerText }) => {
+  return (
+    <div className="flex items-center p-2">
+      <FaCircle className="text-winc-orange mr-2" />
+      <div className="md:text-2xl text-lg"> {innerText} </div>
+    </div>
+  );
+};
+
+const Modal = ({ event, setState }) => {
+  if (!event) return event && <div>No Event</div>;
+  const startTime = new Date(event.start).toLocaleTimeString(
+    navigator.language,
+    {
+      hour: "2-digit",
+      minute: "2-digit",
+    }
+  );
+  const endTime = new Date(event.end).toLocaleTimeString(navigator.language, {
+    hour: "2-digit",
+    minute: "2-digit",
+  });
+  return (
+    <div
+      className={`font-urbanist font-medium fixed top-[56%] left-1/2 -translate-x-1/2 -translate-y-1/2 p-3 w-[75vw] md:w-[50vw] bg-winc-white border-4 border-winc-orange rounded-2xl z-10`}
+    >
+      <div className={`flex justify-between items-center`}>
+        <p
+          className={`text-winc-black bg-winc-orange rounded-full m-0 font-bold py-2 md:py-3 px-3 md:px-4 text-lg md:text-3xl`}
+        >
+          {event.title}
+        </p>
+        <button className="p-0 m-2 text-5xl" onClick={() => setState(null)}>
+          <AiOutlinePlus className="text-winc-black rotate-45 p-0 m-0 hover:text-winc-pink" />
+        </button>
+      </div>
+      <div className="md:text-2xl text-lg p-2">
+        {event.summary.replace(event.summary.split(" ")[0], "")}
```

Comment:
I LOVE the design. The event summary is actually not showing up properly in your ss. The summary is the description we have in dummyEvents.